<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>608</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Components</string>
  </property>
  <widget class="QPushButton" name="deleteUserBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addUserBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showUsersBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>View Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateUserBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Users</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
